samples/resources/disable_manage_resource_deployment_script.py

```diff
@@ -0,0 +1,111 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import os
+
+from azure.core.exceptions import HttpResponseError
+from azure.identity import DefaultAzureCredentials 
+from azure.mgmt.msi import ManagedServiceIdentityClient
+from azure.mgmt.resource import DeploymentScriptsClient, ResourceManagementClient
+
+
+def main():
+
+    SUBSCRIPTION_ID = os.environ.get("SUBSCRIPTION_ID", None)
+    GROUP_NAME = "testgroupx"
+    IDENTITY_NAME = "uai"
+    SCRIPT_NAME = "scriptxx"
+
+    # Create client
```

Comment:
Put a comment here:

For other authentication approaches, please see: https://pypi.org/project/azure-identity/